docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,506 @@
+        placeholder="Search by name or email"
+        size="lg"
+        ref={ref}
+      />
+    )
+  }
+
+  return (
+    <React.Fragment>
+      <AvatarGroup
+        accessibilityLabel="Collaborators: Keerthi, Alberto, and 10 more. Add collaborators to this board."
+        accessibilityExpanded={open}
+        addCollaborators
+        role="button"
+        onClick={() => setOpen( open => !open)}
+        ref={anchorRef}
+        size="md"
+        collaborators={[
+
+          {
+            name: 'Keerthi',
+            src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+          },
+          {
+            name: 'Alberto',
+            src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+          },
+          ...new Array(10),
+        ]}
+        />
+      {open && (
+        <Layer>
+          <Popover
+            anchor={anchorRef.current}
+            idealDirection="down"
+            onDismiss={() => setOpen(false)}
+            positionRelativeToAnchor={false}
+            size="xl"
+          >
+            <Box
+              flex="grow"
+              marginEnd={4}
+              marginStart={4}
+              marginTop={6}
+              marginBottom={8}
+              width={360}
+            >
+              <Flex direction="column" gap={6}>
+                <Text align="center" color="darkGray" weight="bold">
+                  Invite collaborators
+                </Text>
+                <SearchCollaboratorsField />
+              </Flex>
+            </Box>
+          </Popover>
+        </Layer>
+      )}
+    </React.Fragment>
+  );
+}
+  `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(<MainSection name="Localization" description="Be sure to localize  `accessibilityLabel`." />);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="Fixed sizes"
+      description="AvatarGroup is available in 3 fixed height sizes: 'xs' (24px) , 'sm' (32px), and 'md' (48px)."
+    >
+      <CombinationNew size={['xs', 'sm', 'md']}>
+        {({ size }) => (
+          <AvatarGroup
+            accessibilityLabel="Collaborators: Keerthi, Alberto, and Shanice."
+            size={size}
+            collaborators={[
+              {
+                name: 'Keerthi',
+                src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+              },
+              {
+                name: 'Alberto',
+                src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+              },
+              {
+                name: 'Shanice',
+                src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+              },
+            ]}
+          />
+        )}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Responsive sizing"
+      description="AvatarGroup is a responsive component. Avatar Groups that are not given a size prop or use size=`fit` will expand to fit to the width of their parent container. A common use case is to achieve column-based sizing.
```

Comment:
this should either be "size `fit`" or "`size='fit'`", kind of a weird in-between state right now

Reply:
DONE